test-packages/e2e-tests/srv/test-service.js

```diff
@@ -8,6 +8,10 @@ module.exports = async srv => {
     oRequest.reply('SapCloudSdk');
   });
 
+  srv.on('returnSapCloudSdk2', async oRequest => {
+    oRequest.reply('SapCloudSdk');
```

Comment:
minor: This function has two parameters. Maybe we can return some value related to the parameters.